src/main/scala/uk/gov/nationalarchives/Lambda.scala

```diff
@@ -99,15 +99,32 @@ class Lambda extends RequestStreamHandler {
     } yield ()
   }.unsafeRunSync()
 
-  private def generateSlackMessage(folderUpdateRequest: EntityWithUpdateEntityRequest): String = {
-    val entity = folderUpdateRequest.entity
+  private def generateSlackMessage(
+      preservicaUrl: String,
+      folderUpdateRequest: EntityWithUpdateEntityRequest
+  ): String = {
+    val entity: Entity = folderUpdateRequest.entity
+    val entityTypeShort = entity.entityType
+      .map(_.entityTypeShort)
+      .getOrElse("IO") // We need a default and Preservica don't validate the entity type in the url
+    val entityUrl = s"$preservicaUrl/explorer/explorer.html#properties:$entityTypeShort&${entity.ref}"
     val updateEntityRequest = folderUpdateRequest.updateEntityRequest
-    s""":preservica: Entity ${entity.ref} has been updated
-         |*Old title*: ${entity.title.getOrElse("")}
-         |*New title*: ${updateEntityRequest.title}
-         |*Old description*: ${entity.description.getOrElse("")}
-         |*New description*: ${updateEntityRequest.descriptionToChange.getOrElse("")}
-         |""".stripMargin
+    val firstLine =
+      s""":preservica: <$entityUrl|Entity ${entity.ref}> has been updated
```

Comment:
I was thinking that maybe it would be better/cleaner to not hyperlink "Entity" (it also makes it easier to select from the left); what do you think?

Reply:
Do you mean just put the url in without the hyperlink? The url is quite long with a uuid in in it so it's not going to look amazing in the message. We could put "Entity has been updated" in the link if that makes it easier to click.

Reply:
Oh, no, I mean just put "Entity " before the `<` like """":preservica: Entity <$entityUrl|${entity.ref}> has been updated"

Reply:
Ah yeah, that's possible. I've changed it.